Halcyon/Halcyon.Data.Inventory.MySQL/MysqlStorageImpl.cs

```diff
@@ -255,7 +291,7 @@ public InventoryFolderBase getUserRootFolder(UUID user)
                         {
                             rootFolder = items[0];
                         }
-
```

Comment:
Interesting.  Must have run across cases where the `Level` was wrong?

Reply:
It just was never set. Level was something Tranq dreamed up I think, and initialized in Cassandra.  This code never set it, or left it at a default (can't quite remember, same thing either way). So you'll see a couple of new places where I initialize the `Level` to the known correct value. Line 80 does the same thing.